riemann/client.js

```diff
@@ -27,29 +27,29 @@ function _sendMessage(contents, transport) {
     if (transport) {
       transport.send(message);
 
-    // if we're sending a message that is larger than the max buffer
-    // size of UDP, we should switch over to TCP.
+      // if we're sending a message that is larger than the max buffer
+      // size of UDP, we should switch over to TCP.
     } else if (message.length >= MAX_UDP_BUFFER_SIZE) {
       self.tcp.send(message);
 
-    // utilize whatever transport this message is applied to,
-    // by caller.
+      // utilize whatever transport this message is applied to,
+      // by caller.
     } else {
       this.send(message);
     }
 
     if (self.returnPromise) {
-      return e2p.multi(self, [ 'data', 'sent' ]).then(([ param, e ]) => param);
+      return e2p.multi(self, ['data', 'sent']).then(([param, e]) => param);
     }
   };
 }
 
 
 /* some friendly defaults for event,
    in case they went missing. */
-function _defaultValues(payload) {
-  if (!payload.host)  { payload.host = hostname; }
-  if (!payload.time)  { payload.time = new Date().getTime()/1000; }
+function _defaultValues (payload) {
+  if (!payload.host) { payload.host = hostname; }
+  if (!payload.time) { payload.time = Math.round(new Date().getTime() / 1000); }
```

Comment:
The old value was a float, the protobuf file specs it as an int, so it was failing the verify check for serialization on all inputs that attempted to avoid explicitly setting time in the payload.